suspect_resolutions/commit_correlation.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+from sentry.models import CommitFileChange, GroupRelease, ReleaseCommit
+
+
+def is_issue_commit_correlated(resolved_issue_id: int, candidate_issue_id: int) -> bool:
```

Comment:
Instead of passing in the `resolved_issue_id` and `candidate_issue_id` in this function, would it be better to pass in the `Group`s here? That way your function doesn't have to mess around with querying the DB for the `Group` and you can simply use it as is.

Reply:
Since we're querying for `GroupRelease` and not `Group`, I think passing the `id` should be fine